java/src/org/openqa/selenium/os/OsProcess.java

```diff
@@ -44,6 +44,7 @@
 import org.openqa.selenium.io.CircularOutputStream;
 import org.openqa.selenium.io.MultiOutputStream;
 
+@Deprecated
 class OsProcess {
   private static final Logger LOG = Logger.getLogger(OsProcess.class.getName());
```

Comment:
I appreciate your effort here. I have not looked into it in depth so this is purely out of curiosity. Since I understand that using Java 11 APIs may not necessarily be lift and shift. Why was there a need for a new class ExternalProcess? Was it not possible to modify OsProcess methods to use the Java 11 APIs?

Reply:
@pujagani The OsProcess is the internal class, so keeping the API would mean to keep the `CommandLine` alive.
I don't think the `CommandLine` API should be preserved, it mixes the methods to start a new process with controlling a existing process.
We would also need to implement ways to async start a process, what is not supported by the java process builder. These methods would be unused in the current code base, but should be at least be tested.

Reply:
Thank you for explaining that in depth. I appreciate it. I guess even the package name might need renaming at a later point.